Dockerfile

```diff
@@ -1,50 +1,41 @@
-FROM python:3.5
-MAINTAINER Pit Kleyersburg <pitkley@googlemail.com>
-
-# Install dependencies
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends \
-        sudo \
-        tesseract-ocr tesseract-ocr-eng imagemagick ghostscript unpaper \
-    && rm -rf /var/lib/apt/lists/*
-
-# Install python dependencies
-RUN mkdir -p /usr/src/paperless
+FROM alpine:3.7
 WORKDIR /usr/src/paperless
 COPY requirements.txt /usr/src/paperless/
-RUN pip install --no-cache-dir -r requirements.txt
-
 # Copy application
-RUN mkdir -p /usr/src/paperless/src
-RUN mkdir -p /usr/src/paperless/data
-RUN mkdir -p /usr/src/paperless/media
 COPY src/ /usr/src/paperless/src/
 COPY data/ /usr/src/paperless/data/
 COPY media/ /usr/src/paperless/media/
-
+# Set export directory
+ENV PAPERLESS_EXPORT_DIR /export
```

Comment:
Since we are trying to reduce the size of the image, we should also strive to reduce the amount of layers.

I would suggest to combine this and the next `ENV` instruction down to one, see [here](https://github.com/pitkley/docker-paperless/blob/dfbd8864b21d87205b9dc8352deaae123ef3a28b/Dockerfile#L3-L5) for how it works.